control-plane/agents/src/bin/core/controller/scheduling/mod.rs

```diff
@@ -168,6 +168,18 @@ impl ChildSorters {
         match Self::sort_by_health(a, b) {
             Ordering::Equal => match Self::sort_by_child(a, b) {
                 Ordering::Equal => {
+                    // Remove replicas from nodes which are cordoned with most priority.
+                    if let (Some(a), Some(b)) = (a.node_spec(), b.node_spec()) {
+                        match a.cordoned().cmp(&b.cordoned()) {
+                            Ordering::Less => {
+                                return Ordering::Greater;
+                            }
+                            Ordering::Equal => {}
+                            Ordering::Greater => {
+                                return Ordering::Less;
+                            }
+                        }
```

Comment:
```suggestion
                        match b.cordoned().cmp(&a.cordoned()) {
                            Ordering::Equal => {}
                            _else => return _else,
                        }
```

Reply:
Should it not be opposite, the way I did? false is Lesser than true, that means the one with true should be Ordering::Lower to be preffered more?

Reply:
See b swapped with a, so it's the same as what you had, with less lines.

Reply:
Done.